Cargo.toml

```diff
@@ -23,10 +23,11 @@ travis-ci = { repository = "auto-impl-rs/auto_impl" }
 proc-macro = true
 
 [features]
-nightly = ["proc-macro2/nightly"]
+# this feature is obsolete and doesn't mean anything anymore
```

Comment:
Nop, the feature is still used for `def_site` spans. Please:
- remove this comment
- adjust the last doc paragraph in `lib.rs` (line 217 to 224)
- Remove the comment about the nightly feature and better error messages in `examples/error_messages.rs`

Thanks!